test/distributed/org/apache/cassandra/distributed/impl/AbstractCluster.java

```diff
@@ -184,6 +184,12 @@
         private INodeProvisionStrategy.Strategy nodeProvisionStrategy = INodeProvisionStrategy.Strategy.MultipleNetworkInterfaces;
         private ShutdownExecutor shutdownExecutor = DEFAULT_SHUTDOWN_EXECUTOR;
 
+        {
```

Comment:
Is there a reason you went w/the instance initializer vs. static initializer? I'd expect the latter to appropriately nuke the param for everything created inside the JVM context but I could be missing something.

Reply:
No particular reason AFAIR but the cluster initialization seemed more consistent to me that a static initializer because those properties are really relevant for the instance rather than static class. If we ever come up with the ability to pass individual system properties to node instances in JVM dtests, setting properties in a static initializer will become not relevant a all